### PR TITLE
Array Questions

### PR DESCRIPTION
**Pull request by Ayush7614**

### CODE REVIEW

**Comment by Ayush7614** [comment]
Plss review it @smv1999 sir

File: DSA 450 GFG/reverse an array or string.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	// Function calling
+	rvereseArray(arr, 0, n-1);
+	
+	cout << "Reversed array is" << endl;
+	
+	// To print the Reversed array
+	printArray(arr, n);
+	
+	return 0;
+}
```
Comment (smv1999) [inline DSA 450 GFG/reverse an array or string.cpp:47]: Change this file name to reverse_an_array. There is no string involved here.

**Review by smv1999 — CHANGES_REQUESTED** [review]
Just add the problem statements also at the top in comments (for reference).

**Comment by Ayush7614** [comment]
Now @smv1999 sir plss review it

**Review by smv1999 — APPROVED** [review]
Looks good to me :)

**Pull request merged**